src/components/dialogs/voltage-level-creation-dialog.js

```diff
@@ -353,7 +354,7 @@ const VoltageLevelCreationDialog = ({
         formProps: filledTextField,
         values: substationOptions?.sort(compareById),
         allowNewValue: true,
-        getLabel: getId,
+        getLabel: getIdOrSelf,
```

Comment:
leave a comment to explain why we now need to get Id sometimes, and Self other times

Reply:
Done